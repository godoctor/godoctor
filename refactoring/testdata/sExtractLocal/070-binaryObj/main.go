package main

import "fmt"

func main() {
	apple := 10
	orange := 3

	average(apple, orange)
}

func average(a int, b int) {
	if a == b { //<<<<< extractLocal,13,5,13,5,newVar,pass
		fmt.Printf("a*b == %i", a*b)
	}
}
