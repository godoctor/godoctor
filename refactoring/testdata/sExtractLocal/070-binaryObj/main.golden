package main

import "fmt"

func main() {
	apple := 10
	orange := 3

	average(apple, orange)
}

func average(a int, b int) {
	newVar := a
	if newVar == b { //<<<<< extractLocal,13,5,13,5,newVar,pass
		fmt.Printf("a*b == %i", a*b)
	}
}
